varya/assets/css/variables-editor.css

```diff
@@ -125,16 +125,18 @@ body {
 	--list--font-family: var(--global--font-secondary);
 	--definition-term--font-family: var(--global--font-primary);
 	--pullquote--font-family: var(--global--font-primary);
-	--pullquote--font-size: var(--heading--font-size-h4);
-	--pullquote--border-width: 4px;
-	--pullquote--color-border: var(--global--color-foreground);
+	--pullquote--font-size: var(--heading--font-size-h2);
+	--pullquote--font-style: italic;
+	--pullquote--line-height: var(--global--line-height-heading);
```

Comment:
I added two variables here, not entirely sure if they're necessary.

Reply:
Hmmm, I like the `line-height` variable. The italic one might be useful too, but just need to be careful that it doesn’t interfere with any inline-italic rules set by the users (which may be a problem already).